src/views/GovernanceTaskList/index.tsx

```diff
@@ -212,19 +236,23 @@ const GovernanceTaskList = () => {
                       direct you to go through the remaining steps.
                     </p>
                   </TaskListDescription>
-                  <Alert type="info">
-                    <span>
-                      To help with that review, someone from the IT Governance
-                      team will schedule a phone call with you and Enterprise
-                      Architecture (EA).
-                    </span>
-                    <br />
-                    <br />
-                    <span>
-                      After that phone call, the governance team will decide if
-                      you need to go through a full governance process.
-                    </span>
-                  </Alert>
+                  {['INTAKE_DRAFT', 'INTAKE_SUBMITTED'].includes(
+                    status || ''
+                  ) && (
+                    <Alert type="info">
+                      <span>
+                        To help with that review, someone from the IT Governance
```

Comment:
I hate to be a pain, but maybe this should `i18n` too.

On that note - since we only support english, do people have thoughts on the `i18n` folder?

Reply:
Not a pain at all, I think all of the plain text in this file should be in an i18n file but I think it might be better to make a tech debt story for cleaning up them all up at once.

I have done similar stories in the past, i'll make some stories and start tackling them in the next weeks.

Reply:
Looks like Bill is already ahead of me, here's a story that captures the work https://jiraent.cms.gov/browse/EASI-652.